taipy/core/scenario/scenario.py

```diff
@@ -52,7 +54,7 @@ class Scenario(_Entity):
     def __init__(
         self,
         config_id: str,
-        pipelines: List[Pipeline],
+        pipelines: List[Union[PipelineId, Pipeline]],
```

Comment:
Do you think Union[List[PipelineId], List[Pipeline], List[Union[PipelineId, Pipeline]]] would be better? or Union[List[PipelineId], List[Pipeline]]

Reply:
true